Feature#5 initialize project

DerekAyala opened the pull request:
# Description
What did you implemented/Why did you implemented this:
 - The project was created using Ruby on Rails.
 - Installed the dependencies bootstrap and popper.js.
 - Project created with PostgreSQL as main database.
 - Generated the pages_controller and a new page home, changing the main route to this one.

DerekAyala committed:
Ruby On Rails proyect started with PostgreSQL

-Rails added to the proyect

Co-authored-by: Anisbeflo <abejarano0@ucol.mx>
Co-authored-by: MauricioNC <mauricio_navarro0@ucol.mx>

DerekAyala committed:
bootstrap added

-bootsrap , popperjs added with yarn

Co-authored-by: Anisbeflo <abejarano0@ucol.mx>
Co-authored-by: MauricioNC <mauricio_navarro0@ucol.mx>

kevinnio reviewed (dismissed):
Nice.

I know this kata uses Rails 6, but Rails 7 is already out. Maybe you guys would like to try setting up a Rails 7 app as an optional exercise just to see if it's any different.

xFerchoVJ reviewed (dismissed):
LGTM!

DerekAyala commented:
@YilKanda can you give us a review ?

YilKanda commented on app/assets/stylesheets/pages.scss:
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
If you don't occupy this file, you can delete it.

YilKanda commented on app/helpers/pages_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
+end
```
If you don't occupy this file, you can delete it.

YilKanda commented on app/javascript/stylesheets/application.scss:
```diff
@@ -0,0 +1 @@
+@import 'bootstrap'
```
Add a blank line.

YilKanda commented on config/database.yml:
```diff
@@ -0,0 +1,86 @@
+  #schema_search_path: myapp,sharedapp,public
+
+  # Minimum log levels, in increasing order:
+  #   debug5, debug4, debug3, debug2, debug1,
+  #   log, notice, warning, error, fatal, and panic
+  # Defaults to warning.
+  #min_messages: notice
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: kata_07_reloj_checador_k7_n21_t3_test
+
+# As with config/credentials.yml, you never want to store sensitive information,
+# like your database password, in your source code. If your source code is
+# ever seen by anyone, they now have access to your database.
+#
+# Instead, provide the password or a full connection URL as an environment
+# variable when you boot the app. For example:
+#
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# If the connection URL is provided in the special DATABASE_URL environment
+# variable, Rails will automatically merge its configuration values on top of
+# the values provided in this file. Alternatively, you can specify a connection
+# URL environment variable explicitly:
+#
+#   production:
+#     url: <%= ENV['MY_APP_DATABASE_URL'] %>
+#
+# Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
+# for a full overview on how database connection configuration can be specified.
+#
+production:
+  <<: *default
+  database: kata_07_reloj_checador_k7_n21_t3_production
+  username: kata_07_reloj_checador_k7_n21_t3
+  password: <%= ENV['KATA_07_RELOJ_CHECADOR_K7_N21_T3_DATABASE_PASSWORD'] %>
```
Do not upload this file to GitHub, instead you can put an example file: database.yml.example, with the basic configuration to make your project work. This file places it in the .gitignore.

DerekAyala replied:
I added to the .gitignore, is necessary delete this file from the remote repository ?

DerekAyala replied:
or just untrack this file

DerekAyala committed:
Changes requested done

- unnecessary files deleted
- added Blank line at the of the file
- untrack database.yml and added a example file

Co-authored-by: Anisbeflo <abejarano0@ucol.mx>
Co-authored-by: MauricioNC <mauricio_navarro0@ucol.mx>

DerekAyala commented:
@YilKanda could you give us again a review?

YilKanda reviewed (approved):
LGTM!
